### PR TITLE
Towards higher TPS with better error handling - Enhancement

### PR DESCRIPTION
**Pull request by NickSneo**

### CODE REVIEW

**Review by ramkri123 — COMMENTED** [review]
Suggest making the exponential backoff time configurable with a default of 5s

**Review by ramkri123 — APPROVED** [review]

File: erc20-load-test-tool/src/main/java/com/vmware/ethereum/service/WorkloadCommand.java [hunk 1]
```diff
@@ -202,4 +211,56 @@ private void receiptProcessor(
       countDownLatch.countDown();
     }
   }
+
+  @SneakyThrows(InterruptedException.class)
+  private void retryReadCheck(BatchRequest retryBatchRequest, Instant startTime) {
+    BatchResponse receiptBatchResponse = null;
+    boolean success;
+    for (long retry = 1; retry <= 5; retry++) {
```
Comment (dparmar) [inline erc20-load-test-tool/src/main/java/com/vmware/ethereum/service/WorkloadCommand.java:219]: Is this hard-coding of retries intentional?

File: erc20-load-test-tool/src/main/java/com/vmware/ethereum/service/WorkloadCommand.java [hunk 2]
```diff
@@ -202,4 +211,56 @@ private void receiptProcessor(
...
+            // something
+            TransactionReceipt receipt = (TransactionReceipt) receiptResponse.getResult();
+            metrics.recordRead(duration, receipt.getStatus());
+            countDownLatch.countDown();
+          }
+        }
+      }
+
+      if (!success) {
+        if (retry == 5) {
```
Comment (dparmar) [inline erc20-load-test-tool/src/main/java/com/vmware/ethereum/service/WorkloadCommand.java:255]: Hardcoded value

File: erc20-load-test-tool/src/main/java/com/vmware/ethereum/config/Web3jConfig.java [hunk 3]
```diff
@@ -70,5 +70,6 @@ public static class Receipt {
     @NotNull private long interval;
     @NotNull private boolean defer;
     @NotNull private boolean checkWritetxFailed;
+    @Positive private int retryWritetxSleep;
```
Comment (dparmar) [inline erc20-load-test-tool/src/main/java/com/vmware/ethereum/config/Web3jConfig.java:73]: I think Write and Tx should be treated as separate words.

File: erc20-load-test-tool/src/main/java/com/vmware/ethereum/service/WorkloadCommand.java (hunk 1 above)
Comment (NickSneo) [reply erc20-load-test-tool/src/main/java/com/vmware/ethereum/service/WorkloadCommand.java]: made configurable

File: erc20-load-test-tool/src/main/java/com/vmware/ethereum/config/Web3jConfig.java (hunk 3 above)
Comment (NickSneo) [reply erc20-load-test-tool/src/main/java/com/vmware/ethereum/config/Web3jConfig.java]: Changed

**Review by ramkri123 — APPROVED** [review]

**Pull request merged**